assets/webpack.config.js

```diff
@@ -111,6 +111,7 @@ module.exports = function (env) {
         "RELEASE_BLOG_LINK",
         "MAPBOX_PRIVATE_KEY",
         "MAPBOX_STYLE_URL",
+        "HOST_DC_CHARGE_PERCENT",
```

Comment:
dont think we need this since this webpack config is only used for non self-hosted

Reply:
ah! good point.